server/db/seeds/01_seeds.sql

```diff
@@ -96,10 +98,17 @@ INSERT INTO joined_events(user_id, event_id, user_attendance) VALUES
 (2, 9, TRUE);
 
 INSERT INTO comments(user_id, event_id, name, message) VALUES
-(5, 3, 'J-Wings', 'I love this event so much!'),
-(1, 1, 'Alice', 'Great event'),
+(5, 3, 'Jennifer', 'I love this so much!'),
+(1, 1, 'Alice', 'This looks like fun!'),
 (6, 4, 'Boris', 'What time we meeting?'),
-(3, 7, 'Mike', 'I love this place!'),
+(3, 7, 'Mike', 'I think I forgot my car keys, can someone check for me?'),
 (3, 3, 'Michael', 'Where can I park?'),
 (8, 5, 'Jennifer Aniston', 'Awesome guys!'),
-(2, 4, 'Bo', 'see you soon Katie');
\ No newline at end of file
+(2, 4, 'Bethany', 'see you soon Katie');
```

Comment:
if you add any new comments to the db make sure the user has joined this event. only joined users can leave comments

Reply:
Resolved by merging from #80